core/src/main/java/org/infinispan/manager/CacheManagerInfo.java

```diff
@@ -157,6 +170,12 @@ public Json toJson() {
             .set("site_coordinator", isSiteCoordinator())
             .set("site_coordinators_address", getSiteCoordinatorsAddress())
             .set("sites_view", Json.make(getSites()));
+
+     if(isRebalancingEnabled() != null) {
+        result.set("rebalancing_enabled", isRebalancingEnabled());
```

Comment:
I'm not sure the if is necessary. If the value is null, then `Json` will use the json nil object in it's place.

That being said, I see that we omit the key elsewhere ... so maybe better to be consistent.

Reply:
yes, sometimes keys are just not present since it might be a matter of being able to read some properties but not all of them (admin, observer...)
i rather omit and treat the value if it's present. if things are well documented I think it's fine. In any case, the principal consumer of these APIs are our own tools like CLI and Console ...